public/includes/find-house.php

```diff
@@ -0,0 +1,82 @@
+<?php
+require_once('db.php');
+
+function get_states() {
+    $mysqli = db_connection();
+    $sql_query = "SELECT * FROM `states`";
+    $result = mysqli_query($mysqli,$sql_query);
+    $json_response = array();
+
+    if ($result) {
+        $no_of_results = $result->num_rows;
+        if ($no_of_results > 0) {
+            while ($row = mysqli_fetch_assoc($result)) {
+                $json_response[] = $row;
+            }
+            http_response_code(200);
+            echo json_encode($json_response);
+        } else {
+            http_response_code(404);
+            echo json_encode(array("message" => "No state found."));
+        }
+    } else {
+        db_query_error($sql_query);
```

Comment:
Can we add `http_response_code(500)` here for internal server error.

Reply:
the http response is added in the function already `includes/db.php`